modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/test/UploadFileEntryMVCActionCommandTest.java

```diff
@@ -140,6 +192,17 @@ public void testProcessAction() throws Exception {
 			TestPropsValues.getCompanyId());
 
 		Assert.assertEquals(user.getUserId(), dlFileEntry.getUserId());
+
+		ResourcePermission resourcePermission =
+			_resourcePermissionLocalService.getResourcePermission(
+				_company.getCompanyId(), DLFileEntry.class.getName(),
+				ResourceConstants.SCOPE_INDIVIDUAL,
+				String.valueOf(dlFileEntry.getFileEntryId()),
+				RoleLocalServiceUtil.getRole(
+					_company.getCompanyId(), RoleConstants.GUEST
+				).getRoleId());
+
+		Assert.assertEquals(544, resourcePermission.getActionIds());
```

Comment:
@JhossephG what does 544 mean? 

You should be using the `_resourcePermissionLocalService.hasResourcePermission` instead...